libcontainer/container_linux.go

```diff
@@ -43,6 +43,7 @@ type linuxContainer struct {
 	criuVersion          int
 	state                containerState
 	created              time.Time
+	criuFeat             *criurpc.CriuFeatures
```

Comment:
Yes, it does not really belong in the linuxContainer struct. Let me try it as a global variable.